README.md

```diff
@@ -17,35 +26,34 @@ For more information on Magma please visit the [official website](https://magmac
 > :warning: Installing this snap will affect your computer's networking configuration.
 > Make sure it is installed on designated hardware (personal computers are strongly discouraged).
 
-### System requirements
+## System requirements
 
-**Hardware (baremetal strongly recommended)**
+### Hardware (baremetal strongly recommended)
```

Comment:
For some reason this doesn't render properly.

Reply:
I think it renders now, let me know if it works for you.